01-node-tutorial/answers/writeWithPromisesAwait.js

```diff
@@ -0,0 +1,69 @@
+const { writeFile, readFile } = require("fs").promises;
+
+async function writer() {
+  await writeFile(
```

Comment:
This is the part that you want inside of the `try-catch` block. See the third example of the [Reading files with Node.js](https://nodejs.dev/en/learn/reading-files-with-nodejs/) page